src/NewCard/NewCard.stories.js

```diff
@@ -0,0 +1,197 @@
+// @flow
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { text, boolean } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+
+import TripSector, { TripDate } from "../TripSector";
+import TripSegment from "../TripSegment";
+import CarrierLogo from "../CarrierLogo";
+import Stack from "../Stack";
+import Text from "../Text";
+import Button from "../Button";
+import Badge from "../Badge";
+import Clock from "../icons/Clock";
+import CardSection from "./CardSection";
+import * as Icons from "../icons";
+import List, { ListItem } from "../List";
+
+import Card from "./index";
+
+const title = text("Title", "Card with title");
+const description = text("Description", "This is description of the card");
+const sectionTitle = text("SectionTitle", "Section Title");
+const sectionDescription = text("SectionDescription", "Section Description");
+
+storiesOf("NewCard", module)
+  .add("Default", () => {
+    return <Card icon={<Icons.Airplane />} title={title} />;
+  })
+  .add("Card with description", () => {
+    return <Card icon={<Icons.Airplane />} title={title} description={description} />;
+  })
+  .add("Card with actions", () => {
+    return (
+      <Card
+        icon={<Icons.Airplane />}
+        title={title}
+        description={description}
+        actions={<Button size="small">Button</Button>}
+      />
+    );
+  })
+  .add("Card with only section", () => {
+    return (
+      <Card>
+        <CardSection>This is content of card</CardSection>
+      </Card>
+    );
+  })
+  .add("Card with sections", () => {
+    return (
+      <Card>
+        <CardSection title={sectionTitle} description={sectionDescription} />
+        <CardSection title={sectionTitle} description={sectionDescription} />
+        <CardSection title={sectionTitle} description={sectionDescription} />
+      </Card>
+    );
+  })
+  .add(
+    "Card with expandable sections",
```

Comment:
There is bug with double border on the last Section.
<img width="1195" alt="Snímek obrazovky 2019-11-26 v 13 52 44" src="https://user-images.githubusercontent.com/7254437/69635172-1db63880-1054-11ea-9169-849b32739d75.png">